cmd/shipperctl/cmd/clean.go

```diff
@@ -0,0 +1,238 @@
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/bookingcom/shipper/cmd/shipperctl/configurator"
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+	apputil "github.com/bookingcom/shipper/pkg/util/application"
+	"github.com/bookingcom/shipper/pkg/util/filters"
+	releaseutil "github.com/bookingcom/shipper/pkg/util/release"
+)
+
+var (
+	decommissionedClusters []string
+
+	CleanCmd = &cobra.Command{
+		Use:   "clean",
+		Short: "clean Shipper objects",
+	}
+
+	cleanDeadClustersCmd = &cobra.Command{
+		Use:   "decommissioned-clusters",
+		Short: "clean Shipper releases from decommissioned clusters",
+		RunE:  runCleanCommand,
+	}
+
+	CountCmd = &cobra.Command{
+		Use:   "count",
+		Short: "count Shipper releases that are scheduled *only* on decommissioned clusters",
+	}
+
+	countContendersCmd = &cobra.Command{
+		Use:   "contender",
+		Short: "count Shipper *contenders* that are scheduled *only* on decommissioned clusters",
+		RunE:  runCountContenderCommand,
+	}
+
+	countReleasesCmd = &cobra.Command{
+		Use:   "release",
+		Short: "count Shipper *releases* that are scheduled *only* on decommissioned clusters",
+		RunE:  runCountReleasesCommand,
+	}
+)
+
+func init() {
+	// Flags common to all commands under `shipperctl clusters`
+	for _, cmd := range []*cobra.Command{countReleasesCmd, countContendersCmd, cleanDeadClustersCmd} {
```

Comment:
I would use another name here, it's not a big deal but it's the same name as the package :-) so it's a bit confusing.